src/pages/commits/InactiveCommits/index.tsx

```diff
@@ -58,7 +61,8 @@ export const InactiveCommitButton: React.FC<InactiveCommitsProps> = ({
   }
 
   return (
-    <Tooltip
+    <StyledTooltip
+      scrolled={isScrolled ? 1 : 0}
```

Comment:
the only reason I did this is because of this

<img width="400" alt="Screen Shot 2022-02-02 at 10 09 24 AM" src="https://user-images.githubusercontent.com/47064971/152181903-3cc2e658-77b0-47e7-8fa7-b34b3eaad689.png">